src/Symfony/Component/Form/Extension/DependencyInjection/DependencyInjectionExtension.php

```diff
@@ -27,7 +28,7 @@ class DependencyInjectionExtension implements FormExtensionInterface
     private iterable $guesserServices;
 
     /**
-     * @param iterable[] $typeExtensionServices
+     * @param array<string, iterable<int, FormTypeExtensionInterface>> $typeExtensionServices
```

Comment:
```suggestion
     * @param array<string, iterable<FormTypeExtensionInterface>> $typeExtensionServices
```

The class does not really care how the inner iterable is keyed.

Reply:
Done